WordPress/Classes/ViewRelated/Post/Scheduling/SchedulingDatePickerViewController.swift

```diff
@@ -211,5 +211,7 @@ extension DatePickerSheet where Self: UIViewController {
         view.addSubview(stackView)
 
         view.pinSubviewToSafeArea(stackView)
+
+        view.tintColor = UIColor.editorPrimary
```

Comment:
Set the `tintColor` for the calendar available from Post Settings. I wasn't able to find a clean way to change this directly from `PostSettingsViewController.m` and don't believe the calendar is used outside of the editor.

Reply:
I suppose this is relatively safe given the view controller resides within `Post` and is specifically named `SchedulingDate`. As you alluded, this change does inhibit reusing this view controller outside of the editor. I am unsure if that is a big concern currently. It is likely alright to utilize this approach.